po/uk.po

```diff
@@ -610,16 +610,17 @@ msgstr "Команда авторизації"
 
 #: src/Gtk/SettingsDialog.vala:269
 msgid "Done"
-msgstr ""
+msgstr "Готово"
 
 #: src/Gtk/TerminalWindow.vala:150
 msgid "Copy"
-msgstr ""
+msgstr "Копія"
 
 #: src/Gtk/TerminalWindow.vala:158
 msgid ""
 "Copies the entire output buffer, including scrollback, to the clipboard."
 msgstr ""
+«Копіює весь вихідний буфер, включаючи прокручування, до буфера обміну».
```

Comment:
«text».
This syntax fails to build for me. Why is it not
"text."
?
Last time it failed also but I just changed it manually myself and didn't bug you about it.